components/datadog/agent/kubernetes.go

```diff
@@ -0,0 +1,55 @@
+package agent
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/components"
+	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
+)
+
+type KubernetesAgentOutput struct {
+	components.JSONImporter
+
+	AgentInstallName        string `json:"agentInstallName"`
+	AgentInstallNameWindows string `json:"agentInstallNameWindows"`
+}
+
+// DockerAgent is a Docker installer on a remote Host
+type KubernetesAgent struct {
+	pulumi.ResourceState
+	components.Component
+
+	AgentInstallName        pulumi.StringOutput `pulumi:"agentInstallName"`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
	AgentInstallNameLinux        pulumi.StringOutput `pulumi:"agentInstallNameLinux"`
```

Reply:
Feel free to discard, we could have both explicit

Reply:
Sounds good, updated